src/api/app/controllers/webui/project_controller.rb

```diff
@@ -314,12 +314,6 @@ def update_target
   end
 
   def repositories
-    unless @project
-      # TODO support flagdetails for remote instances in the API
-      flash[:error] = "You can't show repositories for remote instances"
-      redirect_to :action => :show, :project => params[:project]
-      return
-    end
```

Comment:
Please add an explanation in the commit message why this got removed would be good.